### PR TITLE
RavenDB-13059 - Add an endpoint to get the server store storage report

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-13059

### CODE REVIEW

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs [hunk 1]
```diff
@@ -196,6 +196,33 @@ public Task Format()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/debug/storage/environment/report", "GET", AuthorizationStatus.ClusterAdmin)]
```
Comment (ayende) [inline src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: I think this should be `Operator`, is there a reason this is cluster admin level?

Comment (ppekrol) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: Need to be moved to StorageHandler

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs [hunk 2]
```diff
@@ -196,6 +196,33 @@ public Task Format()
...
+        public Task EnvironmentReport()
+        {
+            var details = GetBoolValueQueryString("details", required: false) ?? false;
+            var env = ServerStore._env;
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                writer.WriteStartObject();
+                writer.WritePropertyName("Environment");
```
Comment (ppekrol) [inline src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: Use same output format as in StorageHandler.

**Review by ppekrol — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: done

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: done

Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: done

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: done

**Review by ayende — APPROVED** [review]

**Review by ppekrol — APPROVED** [review]

**Pull request merged**